hi-ml-cpath/src/health_cpath/utils/plots_utils.py

```diff
@@ -297,6 +341,23 @@ def save_slide_node_figures(
                     extra_slide_dict,
                 )
 
+    def get_metadata(self, results: ResultsType) -> Optional[List[Any]]:
+        """
+        Get metadata of outputs (validation or test) from slides dataset to stratify plots (e.g PR curve, ROC curve).
```

Comment:
add here in which cases the function returns None?

Reply:
and also say more what's in the list! What can I expect in list item i?

Reply:
Thanks, I have added the description now